public/app/features/panel/panellinks/link_srv.ts

```diff
@@ -305,7 +324,7 @@ export class LinkSrv implements LinkService {
     };
 
     if (replaceVariables) {
-      info.href = replaceVariables(info.href, undefined, VariableFormatID.PercentEncode);
```

Comment:
I see now, got confused with why PercentEncoding here, but I have discovered https://github.com/grafana/grafana/pull/64841 :D


Can you add test to cover this problem?

Reply:
sure, I'll add them today